lib/ReactViewModels/ViewState.js

```diff
@@ -7,6 +7,9 @@ var knockout = require('terriajs-cesium/Source/ThirdParty/knockout');
  * Root of a global view model. Presumably this should get nested as more stuff goes into it. Basically this belongs to
  * the root of the UI and then it can choose to pass either the whole thing or parts down as props to its children.
  */
+
+// mobileViews:'data', 'preview', 'nowViewing', 'search';
+
```

Comment:
you could put this doc on the `mobileView` property itself.  Use the jsdoc `/**` style comments.  There's a Sublime plugin called DocBlockr that makes it easy to write them.